### DISCUSSION TITLE
Renovate seems stuck in the specific repository

### DISCUSSION THREAD

**Discussion by int128** [renovatebot/renovate#28061] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

We use the Renovate hosted app in our private repository, but it has stop working since 2023-11-27.
Specifically,

- Renovate does not create a new pull request.
- Renovate does not update the Dependency Dashboard issue.
- I updated a checkbox in the Dependency Dashboard, but Renovate did not update it after my edit.

This problem occurs in the specific repository. It is working well in other repositories in our organization.

Renovate console shows that "There are no jobs in the repository".

<img width="1182" alt="image" src="https://github.com/renovatebot/renovate/assets/321266/d55ba114-691a-4edc-8041-1a9f13f56e4e">

I clicked "Actions" > "Run Renovate Scan" in the menu, but it shows the error:

> A Renovate job is currently running against this repository. Refresh the recent jobs list to see updates.

How can we recover this?

**Comment by rarkins** [comment]
You can try the Actions button and asking to run on the repo. But we can't help you from our end unless you tell us the repo name or (GitHub) ID

**Reply by int128** [reply]
Thank you for the reply. The repository slug is quipper/terraform.

**Reply by int128** [reply]
@rarkins Could you please check the cause on your end?

**Reply by rarkins** [reply]
I have pinged our backend team to take another look. Was this repository working before? e.g. did it have PRs, Dashboard, etc? Or is this a repository which has never worked? Apparently it shows up as onboarded and activated for us.

**Reply by int128** [reply]
Renovate was working well until 2023-11-27. It created or updated PRs and the Dependency Dashboard everyday. Now it has stopped.

**Reply by rarkins** [reply]
Should be resolved now, please confirm if so!

**Reply by int128** [reply]
Thank you for the quick fix! I confirmed Renovate is working properly now.
